data/publiccloud/log_instance.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
+
+COMMAND=$1
+PROVIDER=$2
+INSTANCE_ID=$3
+HOST=$4
+ZONE=$5
+OUTPUT_DIR=/tmp/log_instance/"$INSTANCE_ID"
+LOCK=${OUTPUT_DIR}/.lock
+LOCK_CNT=${OUTPUT_DIR}/.cnt
+PID_FILE=${OUTPUT_DIR}/pid
+SSH_OPTS="-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o LogLevel=ERROR"
+
+ec2_start_log()
+{
+    inc_unique_counter
+    set +e
+    aws ec2 get-console-output --instance-id "$INSTANCE_ID" | jq -r '.Output' > "${OUTPUT_DIR}/${CNT}_get_console_output_start.log"
```

Comment:
We do not have jq installed. And  a simple `zypper in jq` didn't worked for me. So using `--output text` here.